ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -6146,6 +6146,7 @@ double HqlCppTranslator::getComplexity(IHqlExpression * expr, ClusterType cluste
             complexity = 5;
         break;
     case no_distribute:
+    case no_nwaydistribute:
```

Comment:
Not sure what the complexity implies, but is it really appropriate to be the same as a normal distribute?

Reply:
This code is used to calculate the "complexity" of a query.  I'm not convinced it is very useful, but treating it the same as a distribute is a reasonable assumption.